deployment/quick-start/cluster-configuration.yaml.template

```diff
@@ -17,12 +17,13 @@
 
 
 # If corresponding values aren't be set in the machine list, the default value will be filled in.
-default-machine-properties:
```

Comment:
Where to set the password and user name? I know it's unnecssary for aks. But other platform still need it.

Reply:
The total cluster-configuration.yaml was remove.